src/tests/suite/daos_container.c

```diff
@@ -2443,6 +2443,8 @@ co_api_compat(void **state)
 	daos_handle_t		coh;
 	daos_cont_info_t	info;
 	int			rc;
+	char 			uuid_str1[37];
```

Comment:
(style)  please, no space before tabs

Reply:
The space was there already in several lines above. I just tried to keep the same format locally. Please let me know whether I should remove the extra space around this line. Thank you!